.devcontainer/devcontainer.json

```diff
@@ -2,8 +2,10 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
   "name": "Pycytominer Dev",
-  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
   "image": "mcr.microsoft.com/devcontainers/python:0-3.11",
+  "features": {
+    "ghcr.io/devcontainers-contrib/features/poetry:2": {}
```

Comment:
Nice addition! Consider adding a comment near here to indicate what this does.

Reply:
I think the feature name falls in the category of self-documenting code . So I'd prefer to leave it out.